app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,6 @@
   </head>
 
```

Comment:
- [ ] It would be better if we create a static view for our app, this view will be the root of our app as we have in our project requirements below
![Screenshot from 2023-10-23 22-30-51](https://github.com/Luffytaro22/hello-rails-react/assets/46388113/49624227-c192-4cbe-aef7-94c99a64e36a)
For example; we will create a directory called `static` or `root` or any other name you choose, and in the directory, we create a file `index.html.erb`. The index file should include `<div id="root">`